packages/mendel-exec/index.js

```diff
@@ -136,6 +136,11 @@ module.exports = {
 
                     let normId = parent.deps[depLiteral][runtime];
                     if (typeof normId === 'object') normId = normId[runtime];
+
+                    // If we get _noop from cache, this depLiteral doesn't exist
+                    if (normId === '_noop')
+                        throw new Error(`Cannot find ${depLiteral} from ${mainEntry.id}`); // eslint-disable-line max-len
+
```

Comment:
OK, lets merge this for now.

I still think this is conceptually wrong and I'll open another issue so we eventually improve.

Call stacks are for developers of Mendel, those are client errors detected. I don't think we should be throwing for all this sort of client errors. We should be empowering developers to fix the errors without need to dive into Mendel code.